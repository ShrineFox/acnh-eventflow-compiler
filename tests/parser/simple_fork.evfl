flow Test():
    fork:
        branch:
            TestActor1.Action0()
        branch:
            TestActor2.Action0()
