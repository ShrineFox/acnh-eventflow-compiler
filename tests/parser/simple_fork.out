RootNode[name=Test, vardefs=[], out_edges=[Event4]]
ActionNode[name=Event0, action=EventFlowActionAction0(), params={}, out_edges=[]]
ActionNode[name=Event2, action=EventFlowActionAction0(), params={}, out_edges=[]]
ForkNode[name=Event4, join_node=Event5, out_edges=[Event0, Event2, Event5]]
JoinNode[name=Event5, out_edges=[]]

